package.json

```diff
@@ -197,7 +197,7 @@
     "jest-vue-preprocessor": "^1.5.0",
     "jest-watch-typeahead": "^0.4.0",
     "lerna": "^3.19.0",
-    "lint-staged": "^9.4.1",
+    "lint-staged": "^10.0.6",
```

Comment:
As of `lint-stage` 10 there is no need to add `git add` in tasks definition:
> Prior to version 10, tasks had to manually include git add as the final step. This behavior has been integrated into lint-staged itself in order to prevent race conditions with multiple tasks editing the same files. If lint-staged detects git add in task configurations, it will show a warning in the console. Please remove git add from your configuration after upgrading.

So you can safely removed all of them in: https://github.com/storybookjs/storybook/blob/402c3f7cb1cf090923256b0d8cf6eea93554e428/package.json#L83

Reply:
great!